exercises/concept/tim-from-marketing/.docs/instructions.md

```diff
@@ -0,0 +1,43 @@
+# Instructions
+
+In this exercise you'll be writing code to print name badges for factory employees.
+
+## 1. Print a badge for an employee
+
+Employees have an ID, name and department name. Employee badge labels are formatted as follows: `"[id] - name - DEPARTMENT"`. Implement the (_static_) `Badge.print()` method to return an employee's badge label:
+
+```java
+Badge badge = new Badge();
+badge.print(734, "Ernest Johnny Payne", "Strategic Communication");
+// => "[734] - Ernest Johnny Payne - STRATEGIC COMMUNICATION"
+```
+
+Note that the department should be uppercased on the label.
+
+## 2. Print a badge for a new employee
+
+Due to a quirk in the computer system, new employees occasionally don't yet have an ID when they start working at the factory. As badges are required, they will receive a temporary badge without the ID prefix. Modify the (_static_) `Badge.print()` method to support new employees that don't yet have an ID:
+
+```java
+Badge badge = new Badge();
+Badge.print(id: null, "Jane Johnson", "Procurement");
```

Comment:
Java does not support named parameters, perhaps this was copied directly from the C# track?

```suggestion
Badge.print(null, "Jane Johnson", "Procurement");
```

Reply:
Same for the snippets in the third task, btw.